### ISSUE TITLE
Elm Package Demo Links don't match

### ISSUE THREAD

**Issue by j340m3** [aforemny/material-components-web-elm#84]
When browsing the documentation on [elm packages](https://package.elm-lang.org/packages/aforemny/material-components-web-elm/2.1.1/) some links are dead: [example dead link](https://aforemny.github.io/material-components-web-elm/#text-fields)

**Comment by aforemny** [comment]
Hi @j340m3, I wrote a small script for that. Might be cool to integrate it with Github actions, but I don't have time now to polish it up.

The findings however are here (I did not check them yet):

The following links are 404:
```
https://aforemny.github.io/material-components-web-elm/#checkboxes 404
https://aforemny.github.io/material-components-web-elm/#dialogs 404
https://aforemny.github.io/material-components-web-elm/#drawers 404
https://aforemny.github.io/material-components-web-elm/#elevations 404
https://aforemny.github.io/material-components-web-elm/#fabs 404
https://aforemny.github.io/material-components-web-elm/#icon-buttons 404
https://aforemny.github.io/material-components-web-elm/#image-lists 404
https://aforemny.github.io/material-components-web-elm/#layout-grids 404
https://aforemny.github.io/material-components-web-elm/#menus 404
https://aforemny.github.io/material-components-web-elm/#ripples 404
https://aforemny.github.io/material-components-web-elm/#selects 404
https://aforemny.github.io/material-components-web-elm/#sliders 404
https://aforemny.github.io/material-components-web-elm/#snackbars 404
https://aforemny.github.io/material-components-web-elm/#switches 404
https://aforemny.github.io/material-components-web-elm/#tab-bars 404
https://aforemny.github.io/material-components-web-elm/#text-fields 404
https://aforemny.github.io/material-components-web-elm/#top-app-bars 404
```

The following links are OK:
```
https://aforemny.github.io/material-components-web-elm/#buttons OK
https://aforemny.github.io/material-components-web-elm/#cards OK
https://aforemny.github.io/material-components-web-elm/#chips OK
https://aforemny.github.io/material-components-web-elm/#data-table OK
https://aforemny.github.io/material-components-web-elm/#linear-progress OK
https://aforemny.github.io/material-components-web-elm/#lists OK
https://aforemny.github.io/material-components-web-elm/#radio-buttons OK
https://aforemny.github.io/material-components-web-elm/#theme OK
https://aforemny.github.io/material-components-web-elm/#typography OK
```

**Comment by aforemny** [comment]
I uploaded the script here [check-links.sh](https://github.com/aforemny/material-components-web-elm/blob/master/bin/check-links.sh). GitHub actions should call `sh bin/check-links.sh` after `make`.

**Comment by aforemny** [comment]
Hi @j340m3, this should be fixed in 7a0ed82bf02e3214875ea6e36307c2f56b5eed58. Thanks for reporting it!
